### DISCUSSION TITLE
Everything is mirror, I don't think that User_Setup.h is being read

### DISCUSSION THREAD

**Discussion by ratbasstard** [Bodmer/TFT_eSPI#2383] [Q&A - General]
ESP8266 D1 Mini connected to 1.8 128x160 TFT. I am quite sure the wiring is correct as every Example sketch works - except everything is mirrored.  I have placed  #include <User_Setup.h> into the beginning of the main sketch and placed my User_Setup.h file in the same folder as the sketch - I have tried every TAB option for the ST7735 and nothing seems to have any effect - almost as if the User_Setup.h file is not being read/included when compiling the sketch. I have tried many examples from the 160x128 Example folder. I know I am missing something simple, any suggestions?

**Comment by Bodmer** [comment]
Assuming you are using the Arduino IDE, then the setup in the TFT_eSPI library folder must be edited. Setup files in the sketch folder will NOT work.
